net.sf.eclipsecs.target/net.sf.eclipsecs.target.target

```diff
@@ -1,25 +1,30 @@
 <?xml version="1.0" encoding="UTF-8" standalone="no"?>
 <?pde?>
-<!-- generated with https://github.com/eclipse-cbi/targetplatform-dsl -->
 <target name="Eclipse Checkstyle" sequenceNumber="1653761205">
-  <locations>
-    <location includeMode="slicer" includeAllPlatforms="false" includeSource="true" includeConfigurePhase="true" type="InstallableUnit">
-      <unit id="org.eclipse.jdt.feature.group" version="3.18.500.v20200902-1800"/>
-      <unit id="org.eclipse.sdk.ide" version="4.17.0.I20200902-1800"/>
-      <unit id="org.eclipse.emf.feature.group" version="2.23.0.v20200822-0801"/>
-      <repository location="https://download.eclipse.org/releases/2020-09/202009161000/"/>
-    </location>
-    <location includeMode="slicer" includeAllPlatforms="false" includeSource="true" includeConfigurePhase="true" type="InstallableUnit">
-      <unit id="org.junit.jupiter.api" version="5.6.0.v20200203-2009"/>
-      <unit id="org.junit.jupiter.engine" version="5.6.0.v20200203-2009"/>
-      <unit id="org.junit.jupiter.params" version="5.6.0.v20200203-2009"/>
-      <unit id="org.junit.platform.commons" version="1.6.0.v20200203-2009"/>
-      <unit id="org.junit.platform.engine" version="1.6.0.v20200203-2009"/>
-      <unit id="org.junit.platform.launcher" version="1.6.0.v20200203-2009"/>
-      <unit id="org.junit.platform.runner" version="1.6.0.v20200203-2009"/>
-      <unit id="org.junit.platform.suite.api" version="1.6.0.v20200203-2009"/>
-      <repository location="https://download.eclipse.org/tools/orbit/downloads/drops/R20200831200620/repository"/>
-    </location>
-  </locations>
-  <targetJRE path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-11"/>
-</target>
+	<targetJRE path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-11"/>
+	<locations>
+		<location type="Target" uri="file:${project_loc:/net.sf.eclipsecs.core}/../net.sf.eclipsecs.target/net.sf.eclipsecs.partial.target"/>

```

Comment:
You use a "partial" target file here just for code organization / clarity?

Reply:
Yes. The Maven part of the target must be edited manually, and the remaining Eclipse part can be generated. However, you cannot have the generation and the manual editing in one target file, therefore the generated part is included as "partial". If you think there is a better name for more clarity ("generated" instead?), I'll happily accept every suggestion.

Reply:
Nope, partial sounds good to me. I'm just trying to make sure I understand.
We could take this a step further and make another partial for the maven libs, and the main target file would just be an aggregate of partials.

Reply:
I would be fine with that, too. At work, I use exactly such a 3 target files setup. I wasn't sure if I confuse people even more if I go from 1 to 3 here. :)